tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -447,5 +447,11 @@ public static class Tags
         internal const string AerospikeNamespace = "aerospike.namespace";
 
         internal const string AerospikeSetName = "aerospike.setname";
+
+        internal const string CouchbaseOperationCode = "couchbase.operation.code";
+        internal const string CouchbaseOperationBucket = "couchbase.operation.bucket";
+        internal const string CouchbaseOperationKey = "couchbase.operation.key";
+        internal const string CouchbaseOperationHost = "couchbase.operation.host";
+        internal const string CouchbaseOperationPort = "couchbase.operation.port";
```

Comment:
We could use standard tags like `out.host` and `out.port` which are already defined above.

Reply:
ok fixed